canvas_modules/common-canvas/src/common-properties/controls/expression/expression.scss

```diff
@@ -21,7 +21,7 @@
 	.cm-editor {
 		height: inherit;
 		width: 100%;
-		background: $field-02;
+		background: $field-01;
```

Comment:
What is this changing?

Reply:
This is changing to code editor background previously it was grey `field-02` now I have changed it to white `field-01`.